locallib.php

```diff
@@ -22,6 +22,9 @@
  * @copyright  Accredible <dev@accredible.com>
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
+defined('MOODLE_INTERNAL') || die();
+
+require_once($CFG->dirroot . '/mod/quiz/locallib.php');
```

Comment:
This is needed so the method `quiz_get_best_grade` doesn't fail when calling this method from the external function.